README.md

```diff
@@ -1,24 +1,245 @@
+```sh
+ gem install rails
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone git@github.com:Diegogagan2587/blog-app.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd blog-app
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+### Run tests
+
+Test are under construction, but we expect to have them completed soon!
+<!--
+Example command:
+To run tests, run the following command:
+
+
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+The deployment will be available soon !
+<!--
+Example:
+You can deploy this project using:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Would you like to contact the collaborators of this project?
+
+👤 **Diego Vidal Lopez**
+
+- GitHub: [@Diegogagan2587](https://github.com/Diegogagan2587)
+- Twitter: [@dieg02587](https://twitter.com/dieg02587)
+- LinkedIn: [Diego Vidal Lopez](https://www.linkedin.com/in/diego-vidal-lopez)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> We this project can improve, we are planning to include the next features soon:
+
+- [ ] **User Interface**
+- [ ] **Test for controllers**
+- [ ] **views like: home page, user page, post page**
+- [ ] **Logging form**
+- [ ] **Validations**
+- [ ] **Authoriztion rules**
+- [ ] **API end points**
+- [ ] **API documentation**
+- [ ] **Model Data**
+- [ ] **Model Data**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Diegogagan2587/blog-app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Would you like to help?
+
+If you like this project feel free to leave an start, as well if you have a suggestion feel free to visit issues page.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for suggestion this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> Learn more about FAQ below:
+
+- **How do I fix Linters errors if I modify the project?**
+
+  - for rubocop run
+  ```sh
+  rubocop -A
+  ```
+  - for Stylint run:
+  ```sh
+  npx stylelint "**/*.{css,scss}" --fix
+  ```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Please fix the MIT link for a professional README file

Reply:
@MahmoudBakr23 Thanks for pointing that out! it's fixed now!